Automattic/vip-design-system
# Improve Tabs, Textarea and Input components along with their Stories

## Description

![image](https://user-images.githubusercontent.com/3402/201184091-cf90be61-df40-4035-8331-737435aaedd6.png)


- Fixed the rendered `title` attribute as `aria-label` on the Tablist component
- Added a link and a button as the tabs example so we can ensure focusable elements gets focus when tabbing from the Tab title;
- Moved old form components like Input, Textarea, Label, Radio, and Checkbox to be used from the new `Form.Component` format. The old way still works.
- Added stories to Textarea, Checkbox, Radio
- Improved all Stories in general
- Moved Deprecated components to the end of the list of components
- Changed storybook configuration to list the stories alphabetically 
- Added `Validation` support to Input and Textarea

## Checklist


## Steps to Test — Tabs

1. Pull down PR.
2. `npm run dev`.
3. Open http://localhost:6006/?path=/story/newtabs--default
4. Check the component renders with an `aria-label` attribute instead of `title`
5. Verify when you TAB (Keyboard) from the TAB title, the content gets focused, and you can tab into links, buttons.

## Steps to Test — Stories in general
- Check the Storybook sidebar is not sorted alphabetically
- New stories: Form/Input, Form/Radio, Form/Checkbox, Form/Textarea
- Dialog has a separate section now
